### ISSUE TITLE
Bonus Assignment Deep Q Learning

### ISSUE THREAD

**Issue by ttorkar** [penguinmenac3/robotics-introduction#4]
Hey,

I'm happy to create a deep learning application for this project. Would that be suitable?

**Comment by penguinmenac3** [comment]
Sure, you can write a README in the style of the other tasks.
I am currently writing a Deep Q-Learning application on a branch, however we do not have a deep learning application in development currently.

Would you mind telling what you have in mind as a deep learning application?

**Comment by penguinmenac3** [comment]
I also wonder what framework you would like to use, bare tensorflow or keras?

**Comment by ttorkar** [comment]
No problem, I'm thinking just a simple keras implementation as it's probably easier for beginners. Looking at only using the sensors and game state to create a simple model. Is there an easy way to get the reward variable?

**Comment by penguinmenac3** [comment]
I for my own experiments I just added a "Reward Sensor". I pushed that bit of code.
You can now use the reward sensor just like the distance/front sensor.

(See https://github.com/penguinmenac3/robotics-introduction/commit/72102183812a7ac918d9fcc9c58dd2a4f124c2bc)

**Comment by penguinmenac3** [comment]
Are you still working on the issue or should I write the task myself?

Sorry for the "urge" but we will need a written and reviewed task till in 14-days. (Then our project will be live)

**Comment by penguinmenac3** [comment]
I guess the name fits more since you asked for a reward function.

**Comment by ttorkar** [comment]
Apologies as I thought I had committed this. Files are committed now.

**Comment by penguinmenac3** [comment]
Merged, see commit 852b72dde5e13ec2bfd23c3e0351dc6a05589405
